src/v2/BaseLoginRouter.js

```diff
@@ -133,7 +133,7 @@ export default Router.extend({
           ]
         };
       }
-      // Need to mimic IdxRespones as idx returns raw response at error case
+      // Need to mimic IdxResponses as idx returns raw response at error case
```

Comment:
I've addressed a few typos in files I touched while debugging - none of which should have an impact on functionality.